trace/requirements.txt

```diff
@@ -1,4 +1,6 @@
 google-cloud-trace==0.23.0
 opencensus==0.7.6
 opencensus-ext-stackdriver==0.7.2
+opencensus-ext-flask==0.7.3
```

Comment:
Why are we adding requirements here?

Reply:
Looks like the Kokoro CI doesn't build successfully and it is checking requirements from this file as well as the child's requirements. Only after adding requirements to this file did it passed successfully.

Reply:
This is because you have currently set up this project as s sub-module inside the larger project. You need to move the other sample into it's own folder so these can both exist under `trace` independently.

Reply:
@dukeng @ZiweiZhao what's the status on moving the other samples into their own folder so that we don't have to have this dependency change?

Reply:
I've done moving them to a new folder. Also I've found the reference to the old folder, I'll change the references upon merging. For privacy,  I will send the link to the reference through chat :)